appveyor.yml

```diff
@@ -31,6 +31,7 @@ build_script:
 
     if ($env:APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH -match "^rel-") {
       $env:DT_RELEASE = "True"
+      git checkout $env:APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH$

```

Comment:
It should be just `$env:APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH`, no need for the final `$` character.

Reply:
fixed